tests/integration_tests/scheduler/test_openpbs_driver.py

```diff
@@ -14,15 +14,55 @@ def mock_openpbs(pytestconfig, monkeypatch, tmp_path):
         return
     mock_bin(monkeypatch, tmp_path)
 
-
+from pathlib import Path
+import os
+from textwrap import dedent 
+import stat
+QSTAT_HEADER = (
+    "Job id            Name             User              Time Use S Queue\n"
+    "----------------  ---------------- ----------------  -------- - -----\n"
```

Comment:
In light of https://github.com/equinor/ert/issues/7404, can this be made smarter to easen maintenance?

Reply:
Yes, I moved the constants to conftest